cypress/utils/formFiller.ts

```diff
@@ -110,19 +110,19 @@ export const selectKaistaHaitta = (kaistaHaitta: HANKE_KAISTAHAITTA_KEY) => {
 export const selectKaistanPituusHaitta = (kaistanPituusHaitta: HANKE_KAISTAPITUUSHAITTA_KEY) => {
   cy.get('#kaistaPituusHaitta-toggle-button').click();
   switch (kaistanPituusHaitta) {
-    case HANKE_KAISTAPITUUSHAITTA.YKSI:
+    case HANKE_KAISTAPITUUSHAITTA.EI_VAIKUTA_KAISTAJARJESTELYIHIN:
       cy.get('#kaistaPituusHaitta-item-0').click();
       break;
-    case HANKE_KAISTAPITUUSHAITTA.KAKSI:
+    case HANKE_KAISTAPITUUSHAITTA.KAISTAVAIKUTUSTEN_PITUUS_ALLE_10_METRIA:
       cy.get('#kaistaPituusHaitta-item-1').click();
       break;
-    case HANKE_KAISTAPITUUSHAITTA.KOLME:
+    case HANKE_KAISTAPITUUSHAITTA.KAISTAVAIKUTUSTEN_PITUUS_10_99_METRIA:
       cy.get('#kaistaPituusHaitta-item-2').click();
       break;
-    case HANKE_KAISTAPITUUSHAITTA.NELJA:
+    case HANKE_KAISTAPITUUSHAITTA.KAISTAVAIKUTUSTEN_PITUUS_100_499_METRIA:
       cy.get('#kaistaPituusHaitta-item-3').click();
       break;
-    case HANKE_KAISTAPITUUSHAITTA.VIISI:
+    case HANKE_KAISTAPITUUSHAITTA.VIKAISTAVAIKUTUSTEN_PITUUS_500_METRIA_TAI_ENEMMAN:
```

Comment:
Would it be possible to rename these to for example ALLE_10_METRIA, ALLE_10_M or something like that?

I mean reading HANKE_KAISTAPITUUSHAITTA.x we should be able to know what length is in question.

There is also a typo here.

Reply:
There is a typo in `VIKAISTAVAIKUTUSTEN` but otherwise I do not quite get what you mean? You would prefer to have `M` instead of `METRIA`? Oh, now I see, the whole `KAISTAVAIKUTUSTEN_PITUUS` in names is redundant information?

Reply:
Yes, exactly. I see that as (possibly) redundant and repeating information.

`HANKE_KAISTAPITUUSHAITTA.KAISTAVAIKUTUSTEN_PITUUS_500_METRIA_TAI_ENEMMAN`
vs.
`HANKE_KAISTAPITUUSHAITTA.500_METRIA_TAI_ENEMMAN`

Reply:
You are right. Except that the name cannot start with a number. Maybe underscore like `_500...`? This change should also be done in the backend and API.

Reply:
I'm not sure if the variable name has to match the backend. The value of course has to. 

The underscore looks a bit ugly to my eyes at least. What if these were 

`HANKE_KAISTAPITUUSHAITTA.PITUUS_500_METRIA_TAI_ENEMMAN`

Then there would be no need for the underscore. But as this is a matter of taste, you can ignore this comment. I'll approve the pr.

Reply:
I agree, underscore as a first character is ugly (ESLint agrees too). I will refactor.